### PR TITLE
Alerting: Style nits for the simple query mode

### PR DESCRIPTION
**Pull request by soniaAguilarPeiron**
**What is this feature?**

This PR adds some style nits for the simple mode in the alert rule form.

**Why do we need this feature?**

We are polishing the UI.

**Who is this feature for?**
All users.

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer:**

After the change: 

Screenshot 2024-09-27 at 17 55 45

Screenshot 2024-09-30 at 16 47 36

Screenshot 2024-09-30 at 10 55 16

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx [hunk 1]
```diff
@@ -673,9 +674,24 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
...
         isOpen={showResetModeModal}
-        title="Switching to simple mode"
-        body="The selected queries and expressions cannot be converted to simple mode. Switching will remove them. Do you want to proceed?"
-        confirmText="Yes"
+        title="Deactivate advanced options"
+        body={
+          <div>
+            <Text element="p">
+              <Trans i18nKey="alerting.queryAndExpressionsStep.disableAdvancedOptions.text">
+                The selected queries and expressions cannot be converted. If you choose to deactivate the advanced
```
Comment (brendamuir) [inline public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx:684]:
```suggestion
                The selected queries and expressions cannot be converted. If you deactivate advanced
```

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx [hunk 2]
```diff
@@ -673,9 +674,24 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
...
+                options, your query and condition will be reset to the default settings.
+              </Trans>
+            </Text>
+            <br />
+            <Text element="p">
+              <Trans i18nKey="alerting.queryAndExpressionsStep.disableAdvancedOptions.question">
+                Do you want to deactivate advanced options?
```
Comment (brendamuir) [inline public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx:691]:
```suggestion
                
```

**Review by brendamuir — APPROVED** [review]

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx [hunk 3]
```diff
@@ -673,9 +676,19 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
...
         isOpen={showResetModeModal}
-        title="Switching to simple mode"
-        body="The selected queries and expressions cannot be converted to simple mode. Switching will remove them. Do you want to proceed?"
-        confirmText="Yes"
+        title="Deactivate advanced options"
+        body={
+          <div>
+            <Text element="p">
+              <Trans i18nKey="alerting.queryAndExpressionsStep.disableAdvancedOptions.text">
+                The selected queries and expressions cannot be converted. If you deactivate advanced options, your query
```
Comment (tomratcliffe) [inline public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx:684]: It feels like "cannot be converted" is more vague. Is there a way we can phrase it that keeps that in still?

File: public/app/features/alerting/unified/components/rule-editor/RuleEditorSection.tsx [hunk 4]
```diff
@@ -74,4 +75,8 @@ const getStyles = (theme: GrafanaTheme2) => ({
   fullWidth: css({
     width: '100%',
   }),
+  reverse: css({
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/RuleEditorSection.tsx:78]: Can we get the same results by swapping the actual DOM elements instead? 🤔

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx (hunk 3 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx:684]: you're right, let's put c`onverted to default`

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx [hunk 5]
```diff
@@ -231,11 +242,32 @@ const getStyles = (theme: GrafanaTheme2) => ({
...
+      flex: 1,
+      width: '100%',
+    }),
     header: css({
       background: theme.colors.background.secondary,
       padding: `${theme.spacing(0.5)} ${theme.spacing(1)}`,
       borderBottom: `solid 1px ${theme.colors.border.weak}`,
       flex: 1,
     }),
+    button: css({
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx:258]: I wonder if we can create a separate component from this style? IIRC we're using the same styles in the `features/expressions` components.

**Review by gillesdemey — APPROVED** [review]
Just one nit on DRY components but no blockers

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx (hunk 5 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx:258]: I think it makes sense, as we want to keep consistency and not duplcate code. I will handle this in a separate PR 👍

Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx:258]: [PR](https://github.com/grafana/grafana/pull/94045) created!

**Pull request merged**